scripts/docker.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Build and upload (to Dockerhub) for cli $1 & version $2.
+# run this script with sudo 
```

Comment:
So the script doesn't need to be run as `sudo`, only some individual commands inside of it do.